### DISCUSSION TITLE
Changing the logo in the Link Preview

### DISCUSSION THREAD

**Discussion by Adsecularis** [ctreffe/alfred#272] [Mortimer Q&A]
When I share a link to Mortimer via something like Slack it shows a preview of the Link (atleast I think it's called a preview). 
The problem I have with that is that it shows the logo of the University of Göttingen.

1. How can I change the logo?
2. How can I turn off the preview altogether?

<br>

![Preview](https://github.com/user-attachments/assets/154913b2-bece-436f-a824-4b67672e3b5c)

**Comment by jobrachem** [comment]
Hi @Adsecularis ,

the config.conf has some options for this purpose:

```
# SECTION: layout ------------------------------------------------------
# General configuration of the appearance of your alfred experiment.
# ----------------------------------------------------------------------
[layout]
# Settings for previews on social media services
preview_title = Online-Studie               # Title MAX 35 characters
preview_description = Jetzt mitmachen und Teil der aktuellen Forschung werden. # Description MAX 155 characters
preview_image =                             # Full URL to custom preview image (2063 + 1080 px) (For Facebook and similar)
preview_image_small =                       # Full URL to custom preview image for small versions (300 x 200 px) (for WhatsApp an similar)
```

Have you tried these options already?

**Reply by Adsecularis** [reply]
I just tried them out.

But the results are still unclear, it's a pain to get a social media app to reload a link preview it already has. I needed to use WhatsApp Web and reload the website cache with Control + F5 to see the new preview. And then WhatsApp Web only loads the text preview.
Slack also loads pictures, but the Slack preview cache only clears every 30 minutes and can't be cleared manually.

After all this I could change the text for the preview, but I had no luck changing the logo.
No matter whether I entered a URL to an other server I found on the internet like: https://www.seforanelson.com/wp-content/uploads/2013/03/image-alignment-300x200-1-300x200.jpg
Or referring to a picture on the mortimer server like: _static/img/300x200.jpg

But atleast the preview had no logo at all anymore. Which is good enough for me.

**Reply by jobrachem** [reply]
Thanks for the update. I added an issue for the preview image problem here: https://github.com/ctreffe/alfred/issues/273
